.github/workflows/tests-php.yml

```
@@ -251,40 +251,39 @@ jobs:
+    outputs:
+      matrix: ${{ steps.matrix.outputs.result }}
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@de0fac2e4500dabe0009e67214ff5f5447ce83dd # v6.0.2
+
+      - name: Generate matrix
+        id: matrix
+        env:
+          GH_TOKEN: ${{ github.token }}
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:2:26: Double quote to prevent globbing and word splitting

(shellcheck)